flights/berks_ceh_conditions_flight.yaml

```
@@ -188,7 +188,7 @@ associationDefinitions:
 
   partof1:
     fqn: "ol.exhibits"
-    entitySetName: "BerksCEHHas"
+    entitySetName: "BerksCEHPartOf"
```

For all of the associations, our naming convention has been to just say "Berks" as a prefix because it doesn't seem needed to have separate association datasets for Berks Jail, CEH, HealthChoices, etc. So can you take the "CEH" out of all the association names so they match the other association names in the other flights from jail, etc.? Then it'll read the data into the same datasets. So here it'd be BerksPartOf.
```suggestion
    entitySetName: "BerksPartOf"
```